bazel/scripts/run_integ_tests.sh

```diff
@@ -0,0 +1,175 @@
+#!/usr/bin/env bash
+
+################################################################################
+# Copyright 2022 The Magma Authors.
+
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+################################################################################
+
+set -euo pipefail
+
+###############################################################################
+# FUNCTION DECLARATIONS
+###############################################################################
+
+help() {
+    echo "Executes all bazelified integration tests."
+    echo "Usage:"
+    echo "   $(basename "$0")" 
+    echo "      Execute all tests in the magma repository." 
```

Comment:
Do you really mean "all tests in the magma repository" including all unit/sudo tests?

Reply:
I specified which tests in line 23 and in line 26 I'm referring to all those tests. I see now that this might be confusing.